modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -408,144 +531,210 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::computeStressInitialize(
-                  weights[t] *= -sigmoid(_input_limits[t][in_index][0],
-                                         _input_limits[tt][in_index][1],
-                                         input,
-                                         derivative);
-                else
-                  weights[t] *= (1.0 - sigmoid(_input_limits[t][in_index][0],
-                                               _input_limits[tt][in_index][1],
-                                               input));
+                overlap = true;
+
+                // If current tile is below the second tile's window of applicability
+                if (_input_limits[p][t][in_index][0] < _input_limits[p][tt][in_index][0] &&
+                    _input_limits[p][t][in_index][1] > _input_limits[p][tt][in_index][0])
+                {
+                  weights[p][t] *= sigmoid(_input_limits[p][tt][in_index][0],
+                                           _input_limits[p][t][in_index][1],
+                                           input,
+                                           derivative);
+                }
+                // If current tile is above the second tile's window of applicability
+                else if (_input_limits[p][t][in_index][0] > _input_limits[p][tt][in_index][0] &&
+                         _input_limits[p][t][in_index][0] < _input_limits[p][tt][in_index][1])
+                {
+                  if (derivative)
+                    weights[p][t] *= -sigmoid(_input_limits[p][t][in_index][0],
+                                              _input_limits[p][tt][in_index][1],
+                                              input,
+                                              derivative);
+                  else
+                    weights[p][t] *= (1.0 - sigmoid(_input_limits[p][t][in_index][0],
+                                                    _input_limits[p][tt][in_index][1],
+                                                    input));
+                }
               }
             }
           }
+
+          // If not overlapping, weight is not updated, and the derivative of tile weight is set to
+          // zero
+          if (!overlap && derivative)
+            weights[p][t] = 0.0;
+        }
+      }
+      // If input is below the lower tile limit
+      else if (input < _input_limits[p][t][in_index][0])
+      {
+        // If the lower tile limit equals the lower global limit
+        if (_input_limits[p][t][in_index][0] == _global_limits[in_index][0])
+        {
+          if (_window_failure[in_index][0] == WindowFailure::EXTRAPOLATE)
+          {
+            if (derivative)
+              weights[p][t] *= -sigmoid(0.0, _input_limits[p][t][in_index][0], input, derivative);
+            else
+              weights[p][t] *= (1.0 - sigmoid(0.0, _input_limits[p][t][in_index][0], input));
+            input = _input_limits[p][t][in_index][0];
+          }
+          else if (_window_failure[in_index][0] == WindowFailure::USELIMIT)
```

Comment:
should this set weights=0?  Same with line 670.  Is USELIMIT the same as DONOTHING enum you have commented out?

Reply:
If USELIMIT is used, it just changes the input limit to the edge of the box of applicability, and runs it normally. DONOTHING sets weight to zero.

Reply:
what does USELIMIT set the weights to?  I don't see where weights would be set.

Reply:
Essentially multiplies by 1. You can't set the weights to anything, except zero, only multiple them by the local weight, since it's the accumulation of weights for all input values.